### ISSUE TITLE
itemz does not use new € sign

### ISSUE THREAD

**Issue by sur5r** [nnev/c128-kasse#18]
I just updated the floppy with current master and found `itemz` not to be using the new € sign. I have no time to fix it myself, though.

**Comment by sur5r** [comment]
Maybe this behaves inconsistently for some reason on real hardware. Will have to try.

**Comment by stapelberg** [comment]
Ah, I think the problem might be that the itemz program does not call https://github.com/nnev/c128-kasse/blob/7ac9f791f891aea4a1b00862d56bc3ad90f8db72/src/kasse.c#L307 when starting.

When autostarting the kasse disk image (like we do in `make vice`), the character set is modified and that influences the `itemz` program, too.

But, when changing `-autostart kasse.d71` to `-8 kasse.d71` and running `run"itemz`, I see a broken euro symbol indeed:
![euro](https://user-images.githubusercontent.com/55506/173033315-e525042c-59a3-42ab-91d9-ecb18632e382.jpg)

I have been thinking about merging `itemz` into `kasse`, so that we only need the one program. Was there any reason against that, or have we just never gotten around to doing it? (We did something similar for the credit manager AFAIR.)

**Comment by sur5r** [comment]
I think we simply never got around to it. Also, modifying `items` while running `kasse` could have interesting side effects. The data model also does not cope with prices changing over time. At least the sum is going to be wrong.

**Comment by stapelberg** [comment]
I think commit 62aba6cbe653d049d96266127b63e629fd01346c should do it :)

We’re re-loading the items file after running the `itemz_manager()` function.

**Comment by sur5r** [comment]
Makes sense at a first glance. Should be tested before the event, though.
